dali/operators/image/crop/bbox_crop.cc

```diff
@@ -125,53 +123,57 @@ produce a valid crop to match a single minimum IoU value from `thresholds`. Be a
+The first input, `bboxes`, refers to bounding boxes that are provided as a 2-dimensional tensor where
+the first dimension refers to the index of the bounding box and the second dimension refers to the
 index of the coordinate.
 Coordinates are relative to the original image dimensions (i.e. range [0.0, 1.0]), representing
-either `start` and `end` of the region (e.g. `[left, top, right, bottom]`, or `ltrb`) or `start` and
-`shape` (e.g. `[left, top, width, height]` or `xywh`). In the case of volumetric inputs, bounding
-boxes are encoded as `[left, top, front, right, bottom, back]` or
-`[left, top, front, width, height, depth]`.
+either `start` and `end` of the region or `start` and shape` depending on the value of ``bbox_layout``.
+E.g. ``bbox_layout=\"xyXY\"`` means the bounding box coordinates are following the order
+``start_x, start_y, end_x, end_y``, while ``bbox_layout=\"xyWH\" indicates the order is
+``start_x, start_y, width, height``. See more in ``bbox_layout`` argument description.
 
 A second input `labels` can be optionally provided, representing the labels associated with each of
 the bounding boxes.
 
 Outputs: 0:`anchor`, 1:`shape`, 2:`bboxes`, (3:`labels`,)
 
-The result of the prospective crop operation is provided as two separate outputs `anchor` and
-`shape`, that can be feed directly to `Slice` operator to perform the actual crop operation to the
-original image. `anchor` and `shape` contains starting coordinates and dimensions for the crop, as
+The resulting crop parameters are provided as two separate outputs `anchor` and
+`shape`, that can be fed directly to `Slice` operator to perform the actual cropping of the
+original image. `anchor` and `shape` contain starting coordinates and dimensions for the crop, as
 `[x, y, (z,)]` and `[w, h, (d,)]` respectively. The coordinates can be represented in absolute or
-relative terms, depending of whether we used a fixed `crop_shape` or not.
+relative terms, depending of whether the fixed `crop_shape` was used or not.
 
-Third and fourth outputs correspond to the adjusted bounding boxes and optionally
+The third and fourth output correspond to the adjusted bounding boxes and optionally
 their corresponding labels. Bounding boxes are always specified in relative coordinates.)code")
     .NumInput(1, 2)  // [boxes, labels (optional),]
-    .NumOutput(3)    // [anchor, shape, bboxes, labels (optional),]
-    .AdditionalOutputsFn(
-      [](const OpSpec& spec) {
-        return spec.NumInput() - 1;  // +1 if labels are provided
-      })
+    .InputDox(
+        0, "boxes", "2D TensorList of float",
+        "Relative coordinates of the bounding boxes represented as a 2D tensor where the first "
+        "dimension refers to the index of the bounding box and the second dimension refers to the "
+        "index of the coordinate.")
+    .InputDox(1, "labels", "1D TensorList of integers",
+              "(optional) Labels associated with each of the bounding boxes.")
```

Comment:
I think that all inputs above the minimal number of inputs will be marked as optional automatically by the doc renderer.